code-challenges.js

```diff
@@ -46,4 +87,13 @@ const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
 const cohortList = ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
 // Expected output: ["Delta", "Echo", "Foxtrot"]
 
+// const array345 =(stockArr) =>{
+//     return stockArr.filter((value, index) =>{
+//         return indexOf[3,4,5]  
+//     })
+//  } 
+//   console.log(array345(stockExchange))
+
 // Explain your code:
+//created my function to extrapolate index 3,4,5
+// tried using filter to go thru index of array but know im missing something 
```

Comment:
This is  a tough one.  I understand your thought process, but I think you may be a bit confused on what indexOf does.  Keep working on your vocab and you'll get there!